pkgs/development/python-modules/pip-system-certs/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, wheel
+, git-versioner
+  , wrapt
+}:
+
+buildPythonPackage rec {
+  pname = "pip_system_certs";
+  version = "4.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-245qMTiNl5XskTmVffGon6UnT7ZhZEVv0JGl0+lMNQw=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+    wheel
+    git-versioner
+  ];
+
+  buildInputs = [
+    wrapt
+  ];
```

Comment:
Where is this package `meta`?

Reply:
Thanks for catching all these issues. Will be more careful next time